dora/core/server/worker/src/main/java/alluxio/worker/page/PagedBlockMetaStore.java

```diff
@@ -303,6 +304,20 @@ public PagedBlockMeta commit(long blockId) {
     return blockMeta;
   }
 
+  @Override
+  public PageStoreDir getStoreDirOfFile(String fileId) throws FileDoesNotExistException {
```

Comment:
if the block is a temp one, do we want the caller to know where it is?

Reply:
This is needed for the caller to commit the temp page to be a permanent page. the committing needs to be done on the cache dir.

Reply:
@dbw9580 What do you think?